oteapi/models/mappingconfig.py

```diff
@@ -1,27 +1,28 @@
 """Pydantic Mapping Configuration Data Model."""
-from typing import Annotated, Dict, List, Optional
+from typing import Dict, List, Optional, Tuple
 
-from pydantic import Field, conlist
+from pydantic import Field
 
 from oteapi.models.genericconfig import GenericConfig
 
-SemanticTriple = Annotated[list, conlist(str, min_items=3, max_items=3)]
-
 
 class MappingConfig(GenericConfig):
     """Mapping Strategy Data Configuration."""
 
     mappingType: str = Field(
-        ..., description="Type of registered mapping strategy. E.g., `mapping/demo`."
+        "mapping",
```

Comment:
Why are you putting in a default here? There should _not_ be a default here.
```suggestion
        ...,
```